src/chickens.c

```diff
@@ -147,24 +221,27 @@ const char *init_cpu(void)
     if (part >= MIDR_PART_T8110_BLIZZARD)
         cpufeat_actlr_el2 = true;
 
-    int core = mrs(MPIDR_EL1) & 0xff;
+    if (part >= MIDR_PART_T8101_ICESTORM && part != MIDR_PART_T8301_THUNDER) {
+        int core = mrs(MPIDR_EL1) & 0xff;
 
-    // Unknown, related to SMP?
-    msr(s3_4_c15_c5_0, core);
-    msr(SYS_IMP_APL_AMX_CTL_EL1, 0x100);
+        // Enable IRQs (at least necessary on t600x)
+        // XXX 0 causes pathological behavior in EL1, 2 works.
+        msr(SYS_IMP_APL_SIQ_CFG_EL1, 2);
+        sysop("isb");
 
-    // Enable IRQs (at least necessary on t600x)
-    // XXX 0 causes pathological behavior in EL1, 2 works.
-    msr(SYS_IMP_APL_SIQ_CFG_EL1, 2);
+        // Unknown, related to SMP?
+        msr(s3_4_c15_c5_0, core);
+    }
 
-    sysop("isb");
+    if (part >= MIDR_PART_T8030_LIGHTNING)
+        msr(SYS_IMP_APL_AMX_CTL_EL1, 0x100);
 
```

Comment:
A13 is checked with project triangulation ppl bypass with a dirty cache line. Not very useful though.